### PR TITLE
MODLD-353: Update API to support language

### PR DESCRIPTION
**Pull request by askhat-abishev**

### CODE REVIEW

File: src/main/java/org/folio/linked/data/mapper/kafka/impl/KafkaSearchMessageMapperImpl.java [hunk 1]
```diff
@@ -195,8 +195,13 @@ private List<BibframeContributorsInner> extractContributors(Resource resource) {
...
   private List<BibframeLanguagesInner> extractLanguages(Resource work) {
-    return getPropertyValues(work.getDoc(), LANGUAGE.getValue())
-      .map(p -> new BibframeLanguagesInner().value(p))
+    return work.getOutgoingEdges()
+      .stream()
+      .filter(re -> LANGUAGE.getUri().equals(re.getPredicate().getUri()))
+      .map(ResourceEdge::getTarget)
+      .flatMap(r -> getPropertyValues(r.getDoc(), CODE.getValue())
+        .map(p -> new BibframeLanguagesInner().value(p))
+      )
```
Comment (aleksei-pronichev) [inline src/main/java/org/folio/linked/data/mapper/kafka/impl/KafkaSearchMessageMapperImpl.java:204]: minor
```
.map(r -> r.gtDoc())
.flatMap(doc -> getPropertyValues(doc , CODE.getValue()))
.map(p -> new BibframeLanguagesInner().value(p)))
.toList();
```

**Review by pkjacob — APPROVED** [review]

File: src/main/java/org/folio/linked/data/mapper/kafka/impl/KafkaSearchMessageMapperImpl.java (hunk 1 above)
Comment (pkjacob) [reply src/main/java/org/folio/linked/data/mapper/kafka/impl/KafkaSearchMessageMapperImpl.java:204]: Agree. This is more readable.

Comment (askhat-abishev) [reply src/main/java/org/folio/linked/data/mapper/kafka/impl/KafkaSearchMessageMapperImpl.java]: Done

File: src/main/java/org/folio/linked/data/mapper/dto/common/CoreMapperImpl.java [hunk 2]
```diff
@@ -84,9 +85,10 @@ private <T> T readDoc(JsonNode node, Class<T> dtoClass) {
     try {
       if (nonNull(node)) {
         if (dtoClass == WorkResponse.class) {
-          // Temp fix - targetAudience loaded through the Python ETL have targetAudience in text format
-          // causing the deserialization to fail. Here remove the targetAudience from node
+          // Temp fix - below properties loaded through the Python ETL have values in text format
+          // causing the deserialization to fail. Here remove such properties from node
           ((ObjectNode) node).remove(TARGET_AUDIENCE.getValue());
+          ((ObjectNode) node).remove(LANGUAGE.getValue());
```
Comment (askhat-abishev) [inline src/main/java/org/folio/linked/data/mapper/dto/common/CoreMapperImpl.java:91]: @pkjacob I've added this change also FYI

**Review by PBobylev — APPROVED** [review]

**Pull request merged**